src/app/api/api_v1/endpoints/test.py

```diff
@@ -0,0 +1,19 @@
+from fastapi import APIRouter, Body, HTTPException, Depends
+from sqlalchemy.ext.asyncio import AsyncSession
+from app.api import deps
+
+router = APIRouter()
```

Comment:
For the sake of not indicating a closer grouping of imports with this line than with the top level functions here we should have two newlines between imports and this statement